tasks/github_tasks.py

```diff
@@ -538,3 +538,36 @@ def agenttelemetry_list_change_ack_check(_, pr_id=-1):
             print(
                 "'need-change/agenttelemetry-governance' label found on the PR: potential change to Agent Telemetry metrics is acknowledged and the governance instructions are followed."
             )
+
+
+@task
+def get_required_checks(_, branch: str = "main"):
+    """
+    For this task to work:
+        - A Personal Access Token (PAT) needs the "repo" permissions.
+        - A fine-grained token needs the "Administration" repository permissions (read).
+    """
+    from tasks.libs.ciproviders.github_api import GithubAPI
+
+    gh = GithubAPI('DataDog/datadog-agent')
```

Comment:
```suggestion
    gh = GithubAPI()
```

That's the default value already I think

Reply:
Thanks, fixed with [35be02e](https://github.com/DataDog/datadog-agent/pull/31243/commits/35be02e3bfba8e775ed0836d67cb12ffd774f84d)!